Wabbajack/View Models/UserInterventionHandlers.cs

```diff
@@ -27,13 +26,12 @@ public UserInterventionHandlers(MainWindowVM mvm)
         private async Task WrapBrowserJob(IUserIntervention intervention, Func<WebBrowserVM, CancellationTokenSource, Task> toDo)
         {
             CancellationTokenSource cancel = new CancellationTokenSource();
```

Comment:
This is the code that's misusing NavigateBack, for reference.  Obviously this can be bugfixed, but I left it in to highlight how easily it can be misused.